en/manual/graphics/effects-and-shaders/index.md

```diff
@@ -30,6 +30,9 @@ Stride shaders are converted automatically to the target graphics platform — e
 
 For mobile platforms, shaders are optimized by a GLSL optimizer to improve performance.
 
+>[!NOTE]
+>converting to OpenGL Compute shaders are not supported in SDSL yet.
```

Comment:
Here as well, also the first letter should be capital 🤣🙈

Reply:
oops.. done